draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -2014,24 +2038,24 @@ Input: u, an element of F.
 Output: (x, y), a point on E.
 
 Steps:
-1.   t1 = u^2
-2.   t1 = Z * t1              // Z * u^2
-3.   e1 = t1 == -1            // exceptional case: Z * u^2 == -1
-4.   t1 = CMOV(t1, 0, e1)     // if t1 == -1, set t1 = 0
-5.   x1 = t1 + 1
+1.  tv1 = u^2
+2.  tv1 = Z * tv1             # Z * u^2
+3.   e1 = tv1 == -1           # exceptional case: Z * u^2 == -1
+4.  tv1 = CMOV(tv1, 0, e1)    # if tv1 == -1, set tv1 = 0
+5.   x1 = tv1 + 1
 6.   x1 = inv0(x1)
-7.   x1 = -A * x1             // x1 = -A / (1 + Z * u^2)
-8.  gx1 = x1 + A
+7.   x1 = -C * x1             # x1 = -C / (1 + Z * u^2)
+8.  gx1 = x1 + C
 9.  gx1 = gx1 * x1
-10. gx1 = gx1 + B
-11. gx1 = gx1 * x1            // gx1 = x1^3 + A * x1^2 + B * x1
-12.  x2 = -x1 - A
-13. gx2 = t1 * gx1
+10. gx1 = gx1 + D
+11. gx1 = gx1 * x1            # gx1 = x1^3 + C * x1^2 + D * x1
+12.  x2 = -x1 - C
+13. gx2 = tv1 * gx1
 14.  e2 = is_square(gx1)
-15.   x = CMOV(x2, x1, e2)    // If is_square(gx1), x = x1, else x = x2
-16.  y2 = CMOV(gx2, gx1, e2)  // If is_square(gx1), y2 = gx1, else y2 = gx2
+15.   x = CMOV(x2, x1, e2)    # If is_square(gx1), x = x1, else x = x2
+16.  y2 = CMOV(gx2, gx1, e2)  # If is_square(gx1), y2 = gx1, else y2 = gx2
 17.   y = sqrt(y2)
-18.  e3 = sgn0(u) == sgn0(y)  // Fix sign of y
+18.  e3 = sgn0(u) == sgn0(y)  # Fix sign of y
 19.   y = CMOV(-y, y, e3)
 20. return (x, y)
```

Comment:
here it should be `(X, Y)` ?

Reply:
:+1: pushed a pass over the document that addresses this.